superset-frontend/src/datasource/ChangeDatasourceModal.tsx

```diff
@@ -133,7 +133,7 @@ const ChangeDatasourceModal: FunctionComponent<ChangeDatasourceModalProps> = ({
         }),
       });
     },
-    1000,
+    300,
```

Comment:
just feel curiously, some places we use 200 ms while here we use 300 ms...

Reply:
This place calls a backend API so I set it  longer.